config/datanode.example.toml

```diff
@@ -602,6 +602,9 @@ fork_dictionary_bytes = "1GiB"
 ## Whether to enable the experimental sparse primary key encoding.
 experimental_sparse_primary_key_encoding = false
 
+## The interval to flush the metadata region.
+flush_metadata_region_interval = "30s"
```

Comment:
Is it necessary to expose this config now?

Reply:
I think we should allow metasrv to handle this, but it may be in the next RP.